### ISSUE TITLE
Remove console messages from production

### ISSUE THREAD

**Issue by usernamehw** [phindle/error-lens#18]
```typescript
console.log('Visual Studio Code Extension "errorlens" is now active');
console.log( "Exit onChangedDiagnostics() - !vscode.window" );
console.log( "Exit onChangedDiagnostics() - !activeTextEditor" );
console.log( "Exit updateDecorationsForUri() - uriToDecorate = empty" );
console.log( "Exit updateDecorationsForUri() - !vscode.window" );
console.log( "Exit updateDecorationsForUri() - !activeTextEditor" );
console.log( "Exit updateDecorationsForUri() - activeTextEditor.document.uri.fsPath = empty" );
```

If you really need them then adding logging level from Settings would be nice.

**Comment by phindle** [comment]
Fixed in commit 7d4d211bbe699f8f52b9a66d42686ad8be9fd2a9

**Comment by phindle** [comment]
Fixed in 1.1.3.
